src/OrbitQt/orbitmainwindow.cpp

```diff
@@ -1370,7 +1370,8 @@ std::optional<QString> OrbitMainWindow::LoadSourceCode(const std::filesystem::pa
   return std::nullopt;
 }
 
-void OrbitMainWindow::ShowSourceCode(const std::filesystem::path& file_path, size_t line_number) {
+void OrbitMainWindow::ShowSourceCode(const std::filesystem::path& file_path, size_t line_number,
+                                     std::optional<std::unique_ptr<CodeReport>> maybe_code_report) {
```

Comment:
The name of the parameter should match the name in the interface and in the declaration.

Reply:
Done. Thanks.